tools/ci/check_grep.sh

```diff
@@ -48,10 +50,48 @@ if grep -P '\td[1-2] =' _maps/**/*.dmm;    then
     echo -e "${RED}ERROR: d1/d2 cable variables detected in maps, please remove them.${NC}"
     st=1
 fi;
-echo -e "${BLUE}Checking for stacked cables...${NC}"
-if grep -Pzo '"\w+" = \([^)]*?\n/obj/structure/lattice[/\w,\n]*?[^)]*?\n/obj/structure/lattice[/\w,\n]*?[^)]*?\n/area/.+?\)' _maps/**/*.dmm;	then
+
+# This section checks to make sure only one of any type and its decendant subtypes exists on a tile at a time.
+ONLY_ONE_INCLUDE_SUBTYPE = ( "/obj/structure/lattice", "/obj/machinery/power/apc", "/obj/machinery/airalarm", "/obj/machinery/firealarm", "/obj/effect/spawner/structure/window")
```

Comment:
```suggestion
ONLY_ONE_INCLUDE_SUBTYPE = ( 
    "/obj/structure/lattice"
    "/obj/machinery/power/apc"
    "/obj/machinery/airalarm"
    "/obj/machinery/firealarm"
    "/obj/effect/spawner/structure/window"
)
```

Bash arrays do not use commas.

Reply:
This is a web suggestion, so, spacing could be off.